templates/default/nginx_ice_site.erb

```diff
@@ -14,6 +14,6 @@ server {
   }
 
   location ~ /current/ {
-    rewrite ^/current/(.*)$ /$1 redirect;
+  rewrite ^/current/(.*)$ <%= node['ice']['nginx_rewrite_http_scheme'].nil? ? '' : "#{node['ice']['nginx_rewrite_http_scheme']}://$host" %>/$1 redirect;
```

Comment:
My 0.02 - you'd be better off declaring a default attribute value in default.rb, then getting rid of the `nil` check here. It makes the template more readable, and also gives clarity as to the available configuration options simply by scanning the default attributes list.